model/actor.gd

```diff
@@ -71,6 +74,18 @@ func step_time():
 	if can_draw():
 		draw_cooldown -= draw_rate
 
+func set_upgrade(upgrade):
+	if upgrades.size() == upgrade_slot:
+		upgrades.pop_front();
```

Comment:
I asked you at dircord about it....
I thought that is was possible to renew the upgrades until we make a better management upgrade system (or interface)